libsql-remote/src/cloudflare.rs

```diff
@@ -0,0 +1,76 @@
+use crate::connection::Connection;
+use crate::proto::Stmt;
+use crate::{Error, HttpSend, IntoParams, Result, Rows, ServerMsg, Statement};
+use futures::future::LocalBoxFuture;
+
+#[derive(Debug, Copy, Clone)]
+pub struct CloudflareSender;
+
+impl CloudflareSender {
+    async fn send(url: String, auth: String, body: String) -> Result<ServerMsg> {
+        use worker::*;
+
+        let mut response = Fetch::Request(Request::new_with_init(
+            &url,
+            &RequestInit {
+                body: Some(wasm_bindgen::JsValue::from_str(&body)),
+                headers: {
+                    let mut headers = Headers::new();
+                    headers.append("Authorization", &auth).ok();
```

Comment:
If this code fails, wouldn't it just return HTTP Unauthorized on request send?

Reply:
Yeah, though that could mean both failed to set token or some other reason like a key revoked. We probably want to differentiate. I think a panic is okay since it should be caught quite quickly, but we can also push it to a proper error.